ide-support/revsaveasstandalone.livecodescript

```diff
@@ -928,14 +928,15 @@ command revSaveAsLinuxStandalone pStack, pRestoreFileName, pFolder, @xStandalone
       put tStandaloneSettingsA into sStandaloneSettingsA
       
       local tArch
+      set the itemDelimiter to comma
```

Comment:
I suggest moving this after the extraction of tStackPath above so the change from the default item delimiter is localized to that bit of code.